x/auth/client/cli/query.go

```diff
@@ -35,232 +28,6 @@ const (
 	EventFormat = "{eventType}.{eventAttribute}={value}"
 )
 
-// GetQueryCmd returns the transaction commands for this module
-func GetQueryCmd(ac address.Codec) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:                        types.ModuleName,
-		Short:                      "Querying commands for the auth module",
-		DisableFlagParsing:         true,
-		SuggestionsMinimumDistance: 2,
-		RunE:                       client.ValidateCmd,
-	}
-
-	cmd.AddCommand(
-		GetAccountCmd(ac),
-		GetAccountAddressByIDCmd(),
-		GetAccountsCmd(),
-		QueryParamsCmd(),
-		QueryModuleAccountsCmd(),
-		QueryModuleAccountByNameCmd(),
-	)
-
-	return cmd
-}
-
-// QueryParamsCmd returns the command handler for evidence parameter querying.
-func QueryParamsCmd() *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "params",
-		Short: "Query the current auth parameters",
-		Args:  cobra.NoArgs,
-		Long: strings.TrimSpace(`Query the current auth parameters:
-
-$ <appd> query auth params
-`),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientQueryContext(cmd)
-			if err != nil {
-				return err
-			}
-
-			queryClient := types.NewQueryClient(clientCtx)
-			res, err := queryClient.Params(cmd.Context(), &types.QueryParamsRequest{})
-			if err != nil {
-				return err
-			}
-
-			return clientCtx.PrintProto(&res.Params)
-		},
-	}
-
-	flags.AddQueryFlagsToCmd(cmd)
-
-	return cmd
-}
-
-// GetAccountCmd returns a query account that will display the state of the
-// account at a given address.
-func GetAccountCmd(ac address.Codec) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "account [address]",
-		Short: "Query for account by address",
-		Args:  cobra.ExactArgs(1),
-		RunE: func(cmd *cobra.Command, args []string) error {
-			clientCtx, err := client.GetClientQueryContext(cmd)
-			if err != nil {
-				return err
-			}
-			_, err = ac.StringToBytes(args[0])
-			if err != nil {
-				return err
-			}
-
-			queryClient := types.NewQueryClient(clientCtx)
-			res, err := queryClient.Account(cmd.Context(), &types.QueryAccountRequest{Address: args[0]})
-			if err != nil {
-				node, err2 := clientCtx.GetNode()
-				if err2 != nil {
-					return err2
-				}
-				status, err2 := node.Status(context.Background())
-				if err2 != nil {
-					return err2
-				}
-				catchingUp := status.SyncInfo.CatchingUp
-				if !catchingUp {
-					return errorsmod.Wrapf(err, "your node may be syncing, please check node status using `/status`")
-				}
-				return err
-			}
```

Comment:
@julienrbrt , I always had a doubt how are these cases going to be handled with autoCLI?

Reply:
This command was the only one in this SDK having this check. Given AutoCLI simply just uses gRPC, the CLI has the same behavior as the gRPC endpoints (in this case querying `/cosmos/auth/v1beta1/accounts/<addr>`).